test/functional/wallet_signer.py

```diff
@@ -65,23 +65,22 @@ def run_test(self):
     def test_valid_signer(self):
         self.log.debug(f"-signer={self.mock_signer_path()}")
 
-        # Create new wallets for an external signer.
```

Comment:
In 08f7813f536c242b7a4b65d01cfbc73601846a25 "wallet: make watch-only optional for external signer"

This comment can stay?

Reply:
Brought it back.